lib/src/linden/xstyles.dart

```diff
@@ -128,6 +128,9 @@ class XStyles {
   TextStyle? get textInputTextSmall =>
       textInputText?.copyWith(fontSize: 13.0, height: 1.75);
 
+  TextStyle? get textInputHintSmall =>
+      textInputTextSmall?.copyWith(color: colors.hintTextStyle);
+
```

Comment:
❓  hmm just a question because I see it the first time, why are all TextStyles nullable?

Reply:
discussed, can be closed